src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1975,54 +1975,79 @@ limitations under the License.
             </include>
             <include cluster="Media Playback" client="false" server="true" clientLocked="false" serverLocked="true"></include>
             <include cluster="Keypad Input" client="false" server="true" clientLocked="false" serverLocked="true"></include>
-            <include cluster="Application Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Media Input" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="On/Off" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Channel" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Audio Output" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Low Power" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Wake on LAN" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Target Navigator" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Account Login" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Content Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
+            <include cluster="Application Launcher" client="false" server="true" clientLocked="false" serverLocked="true"></include>
```

Comment:
Why is "Application Launcher" serverLocked=true?  It's an optional cluster for this device type, no?

Reply:
Updated.